src/tye_lab_to_nwb/neurotensin_valence/behavior/neurotensindeeplabcutdatainterface.py

```diff
@@ -0,0 +1,117 @@
+
+    def __init__(
+        self,
+        file_path: FilePathType,
+        config_file_path: FilePathType,
+        verbose: bool = True,
+    ):
+        """
+        Interface for writing pose estimation data from CSV and pickle to NWB.
+
+        Parameters
+        ----------
+        file_path : FilePathType
+            path to the CSV file output by DeepLabCut GUI.
+        config_file_path : FilePathType
+            path to .pickle config file output by DeepLabCut GUI.
+        verbose: bool, default: True
+            controls verbosity.
+        """
+        self.verbose = verbose
+        super().__init__(file_path=file_path, config_file_path=config_file_path)
+
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        pose_estimation_metadata = load_dict_from_file(
+            file_path=Path(__file__).parent.parent / "metadata" / "pose_estimation_metadata.yaml"
+        )
+        metadata.update(pose_estimation_metadata)
+
+        return metadata
+
+    def _load_source_data(self):
+        pose_estimation_data = pd.read_csv(self.source_data["file_path"], header=[0, 1, 2])
+        # The first level contains the scorer, can be dropped from header (can be found also in conf)
+        pose_estimation_data.columns = pose_estimation_data.columns.droplevel(0)
+        pose_estimation_config = pd.read_pickle(self.source_data["config_file_path"])
+        return pose_estimation_data, pose_estimation_config
+
+    def run_conversion(
+        self,
+        nwbfile_path: OptionalFilePathType = None,
+        nwbfile: Optional[NWBFile] = None,
+        metadata: Optional[dict] = None,
+        original_video_file_path: OptionalFilePathType = None,
+        labeled_video_file_path: OptionalFilePathType = None,
+        edges: Optional[ArrayType] = None,
+        column_mappings: Optional[dict] = None,
+        overwrite: bool = False,
+    ):
+        with make_or_load_nwbfile(
+            nwbfile_path=nwbfile_path, nwbfile=nwbfile, metadata=metadata, overwrite=overwrite, verbose=self.verbose
+        ) as nwbfile_out:
+            pose_estimation_data, pose_estimation_config = self._load_source_data()
+
+            pose_estimation_metadata = metadata["PoseEstimation"]
+
+            pose_estimation_series = []
+            for column_name in pose_estimation_metadata:
+                pose_estimation_series_data = pose_estimation_data[column_name]
+
+                pose_estimation_series.append(
+                    PoseEstimationSeries(
+                        name=pose_estimation_metadata[column_name]["name"],
+                        description=pose_estimation_metadata[column_name]["description"],
+                        data=pose_estimation_series_data[["x", "y"]].values,
+                        unit="pixels",
+                        reference_frame="(0,0,0) corresponds to bottom left corner of the cage.",
```

Comment:
```suggestion
                        reference_frame="(0,0) corresponds to bottom left corner of the cage.",
```

Was this confirmed with respect to the port orientation? I.e., if the port is in the bottom right, the subject (x,y) position  are both positive, traversing the positive x and y axes

Reply:
Plotting the port coordinates: 
![port](https://user-images.githubusercontent.com/24475788/221549773-7e5f3b0f-8cc4-4120-95e2-9f5f4399b726.png)
and the camera position that should correspond to "top left":
![topleft](https://user-images.githubusercontent.com/24475788/221550300-98f1c422-470c-4997-98c9-dfc0a3c0a02f.png)
and the "bottom right":
![botright](https://user-images.githubusercontent.com/24475788/221550707-6fee5e04-6d59-4c51-963e-bb81c59f68c3.png)

These are the original values from the CSV, however it seems like the "y" axis should be flipped?

Original positions:
![orig_visual_points](https://user-images.githubusercontent.com/24475788/221549812-88ac5dc7-0616-4173-b927-17bd8e477b8e.png)

Flipped "y" axis:
![visual_points](https://user-images.githubusercontent.com/24475788/221549837-223adca0-ffe1-4008-87df-24f40b84858e.png)
Snapshot from the labelled video:
<img width="631" alt="Screenshot 2023-02-27 at 12 21 52" src="https://user-images.githubusercontent.com/24475788/221551129-72c810c7-642a-44b6-99ac-395a880d8596.png">

Reply:
@CodyCBakerPhD 

The positions in the CSV can be overlayed on the labeled video, the unlabelled video has a different length (num_frames=339811) compared to the labeled video (num_frames=169907):

![points50](https://user-images.githubusercontent.com/24475788/222437633-c2ee2466-c3e4-41e7-b861-506b1e3e787c.png)


![points500](https://user-images.githubusercontent.com/24475788/222437258-41dfb0ae-3b28-40bc-8041-76934a05b9b6.png)

![points5000](https://user-images.githubusercontent.com/24475788/222437776-9c0f3dd3-46cb-4b03-bc4f-b066725a85ae.png)


![points20000](https://user-images.githubusercontent.com/24475788/222437346-2cef2fc0-5035-4f83-80da-18c715dc5017.png)

`(0,0)` shows up in the top left corner when drawing it on the frame.

Reply:
> Was this confirmed with respect to the port orientation? I.e., if the port is in the bottom right, the subject (x,y) position are both positive, traversing the positive x and y axes

```suggestion
                        reference_frame="(0,0) corresponds to the top left corner of the cage.",
```